src/main/resources/db/changelog/changes/v1.0.0/2024-07-22-create-mediated_request-table.xml

```diff
@@ -0,0 +1,182 @@
+            <column name="requester_id" type="uuid">
+                <constraints nullable="false"/>
+            </column>
+            <column name="requester_first_name" type="varchar(255)"/>
+            <column name="requester_last_name" type="varchar(255)"/>
+            <column name="requester_middle_name" type="varchar(255)"/>
+            <column name="requester_barcode" type="varchar(255)"/>
+            <column name="proxy_user_id" type="uuid"/>
+            <column name="proxy_first_name" type="varchar(255)"/>
+            <column name="proxy_last_name" type="varchar(255)"/>
+            <column name="proxy_middle_name" type="varchar(255)"/>
+            <column name="proxy_barcode" type="varchar(255)"/>
+            <column name="instance_id" type="uuid">
+                <constraints nullable="false"/>
+            </column>
+            <column name="instance_title" type="varchar(255)"/>
+            <column name="holdings_record_id" type="uuid"/>
+            <column name="item_id" type="uuid"/>
+            <column name="item_barcode" type="varchar(255)"/>
+            <column name="mediated_workflow" type="varchar(255)"/>
+            <column name="mediated_request_status" type="MediatedRequestStatus">
+                <constraints nullable="false"/>
+            </column>
+            <column name="mediated_request_step" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="status" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="cancellation_reason_id" type="uuid"/>
+            <column name="cancelled_by_user_id" type="uuid"/>
+            <column name="cancellation_additional_information" type="text"/>
+            <column name="cancelled_date" type="timestamp without time zone" defaultValueComputed="now()"/>
+            <column name="position" type="int"/>
+            <column name="fulfillment_preference" type="FulfillmentPreference">
+                <constraints nullable="false"/>
+            </column>
+            <column name="delivery_address_type_id" type="uuid"/>
+            <column name="pickup_service_point_id" type="uuid"/>
+
+            <!-- Search index -->
+            <column name="call_number" type="varchar(255)"/>
+            <column name="call_number_prefix" type="varchar(255)"/>
+            <column name="call_number_suffix" type="varchar(255)"/>
+            <column name="full_call_number" type="varchar(255)"/>
+            <column name="shelving_order" type="varchar(255)"/>
+            <column name="pickup_service_point_name" type="varchar(255)"/>
+
+            <!-- Metadata -->
+            <column name="created_date" type="timestamp" defaultValueComputed="now()"/>
+            <column name="created_by_user_id" type="uuid"/>
+            <column name="created_by_username" type="varchar(100)"/>
+            <column name="updated_date" type="timestamp"/>
```

Comment:
Why we use `timestamp` here but `timestamp without time zone` in other places?

Reply:
Fixed. `timestamp without time zone` makes more sense because all of our dates in the DB are in UTC